.github/workflows/sync-examples.yml

```
@@ -1,7 +1,12 @@
 name: Sync example notebooks from main PGM repo to PGM-workshop
 
 on:
+  push:
+    branches:
+      - sync-examples-workflow
   workflow_dispatch:
+  schedule:
+    - cron: "32 14 * * *" # Based on UTC time
```

```suggestion
    - cron: "32 2 * * *" # Based on UTC time
```

2 AM instead of 2 PM. Typically, syncing logic is done at night to avoid interfering with EU and US peak (work) hours.